coderd/autobuild/lifecycle_executor.go

```diff
@@ -323,14 +324,15 @@ func (e *Executor) runOnce(t time.Time) Stats {
 					}
 				}
 				if shouldNotifyDormancy {
+					dormantTime := time.Now().Add(time.Duration(tmpl.TimeTilDormant))
```

Comment:
One thing to check: is the dormancy value in the database in UTC? If so, the timezone of the control plane might make things weird and we may want to add a `.UTC()` call in here too or use `dbtime.Now()`

Reply:
Good to know